doc/api/tls.md

```diff
@@ -805,6 +856,21 @@ and their processing can be delayed due to packet loss or reordering. However,
 smaller fragments add extra TLS framing bytes and CPU overhead, which may
 decrease overall server throughput.
 
+### tlsSocket.verifyError()
```

Comment:
Sorry if these questions are naïve or otherwise ill-informed: Is this the best name for this? I know it's what we used previously, but it wasn't exposed publicly, was it? Might `validatePeerCertificate() or something like that be a better name?